internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,179 @@
+    $SPLUNK_MEMORY_TOTAL_MIB = "512"
+    write-host "The SPLUNK_MEMORY_TOTAL_MIB parameter is not speacified. Using default configuration."
+}
+
+if (!$SPLUNK_BUNDLE_DIR) {
+    $SPLUNK_BUNDLE_DIR = "$installation_path\agent-bundle"
+    write-host "The SPLUNK_BUNDLE_DIR parameter is not speacified. Using default configuration."
+}
+
+if (!$install_dir) {
+    $install_dir = $installation_path
+    write-host "Setting installation directory to $install_dir"
+}
+
+# remove orphaned service or when upgrading from bundle installation
+try {
+    stop_service
+} catch {
+    write-host "$_"
+}
+
+# remove orphaned registry entries or when upgrading from bundle installation
+try {
+    remove_otel_registry_entries
+} catch {
+    write-host "$_"
+}
+
+update_registry -path "$regkey" -name "SPLUNK_API_URL" -value "$SPLUNK_API_URL"
+update_registry -path "$regkey" -name "SPLUNK_BUNDLE_DIR" -value "$SPLUNK_BUNDLE_DIR"
+update_registry -path "$regkey" -name "SPLUNK_HEC_TOKEN" -value "$SPLUNK_HEC_TOKEN"
+update_registry -path "$regkey" -name "SPLUNK_HEC_URL" -value "$SPLUNK_HEC_URL"
+update_registry -path "$regkey" -name "SPLUNK_INGEST_URL" -value "$SPLUNK_INGEST_URL"
+update_registry -path "$regkey" -name "SPLUNK_MEMORY_TOTAL_MIB" -value "$SPLUNK_MEMORY_TOTAL_MIB"
+update_registry -path "$regkey" -name "SPLUNK_REALM" -value "$SPLUNK_REALM"
+update_registry -path "$regkey" -name "SPLUNK_TRACE_URL" -value "$SPLUNK_TRACE_URL"
+
+$packageArgs = @{
+    packageName    = $env:ChocolateyPackageName
+    fileType       = 'msi'
+    file           = Join-Path "$toolsDir" "MSI_NAME"  # replaced at build time
+    softwareName   = $env:ChocolateyPackageTitle
+    checksum64     = "MSI_HASH"  # replaced at build time
+    checksumType64 = 'sha256'
+    silentArgs     = "/qn /norestart /l*v `"$($env:TEMP)\$($packageName).$($env:chocolateyPackageVersion).MsiInstall.log`" INSTALLDIR=`"$($install_dir)`""
+    validExitCodes = @(0)
+}
+
+Install-ChocolateyInstallPackage @packageArgs
+
+if ($MODE -eq "agent" -or !$MODE) {
+    if (-NOT (Test-Path -Path "$config_path\agent_config.yaml")) {
+        write-host "Copying agent_config.yaml to config_path"
+        Copy-Item "$installation_path\agent_config.yaml" "$config_path"
+    }
+    $config_path = "$program_data_path\agent_config.yaml"
+}
+elseif ($MODE -eq "gateway"){
+    if (-NOT (Test-Path -Path "$config_path\gateway_config.yaml")) {
+        write-host "Copying gateway_config.yaml to config_path"
+        Copy-Item "$installation_path\gateway_config.yaml" "$config_path"
+    }
+    $config_path = "$program_data_path\gateway_config.yaml"
+}
+
+update_registry -path "$regkey" -name "SPLUNK_CONFIG" -value "$config_path"
+
+if (!$SPLUNK_ACCESS_TOKEN) {
+    write-host ""
+    write-host "*NOTICE*: SPLUNK_ACCESS_TOKEN environment variable needs to specify with a valid value. After specifying it to start the splunk-otel-collector service rebooting the system or run the following command in a PowerShell terminal:"

```

Comment:
Not sure this is valid content but something like the following would be clearer imo:

```suggestion
    write-host "*NOTICE*: SPLUNK_ACCESS_TOKEN not detected. This is required for the default configuration to reach Splunk Observability Suite and can be specified via
    write-host "Set-ItemProperty -path `"HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment`" -name `"SPLUNK_ACCESS_TOKEN`" -value `"ACTUAL_ACCESS_TOKEN`"
    write-host "before starting the splunk-otel-collector service:"
```